### ISSUE TITLE
Feature-request: Add tidy method for car::leveneTest

### ISSUE THREAD

**Issue by mkirzon** [tidymodels/broom#732]
It'd be helpful to support `car::leveneTest`, returning a tidy broom dataframe.

**Comment by alexpghayes** [comment]
This would be great! The difficulty is that `leveneTest` returns an `anova` object. **INFINITE SCREAMING**. 

``` r
library(car)
#> Loading required package: carData
library(broom)

lt <- with(Moore, leveneTest(conformity, fcategory))

class(lt)  # ETERNAL SCREAMING
#> [1] "anova"      "data.frame"

tidy(lt)
#> # A tibble: 2 x 4
#>   term     df statistic p.value
#>   <chr> <int>     <dbl>   <dbl>
#> 1 group     2    0.0460   0.955
#> 2 ""       42   NA       NA
```

<sup>Created on 2019-08-07 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

I previously asked John Fox, the maintainer of the `car` package, to give `leveneTest` objects a `leveneTest` class to enable dispatch. He said no.

I suppose we could inspect the title of the `anova` object to detect `leveneTest` results, but that starts to get a bit fragile.

TL; DR: It could probably be done with some effort. I won't be able to get around to it soon but I can review PRs if someone else would like to take it on.

**Comment by mkirzon** [comment]
Ah I see... I saw a related comment on the Manova request. If I wanted to take it on - and I haven't contributed to tidyverse yet - what's your recommendation for where to get started (eg a good broom function as a template, documentation, etc)? I've previously looked at working on `dbplyr` but was quickly overwhelmed without documentation.

**Comment by alexpghayes** [comment]
I would start by writing a function that does the tidying the way you want. Then I would look at https://github.com/tidymodels/broom/blob/master/R/betareg-tidiers.R as a template. There's some documentation at https://broom.tidyverse.org/articles/adding-tidiers.html.

In terms of tidying the leveneTest results, I would try to mirror the behavior of the `htest` tidiers to a large degree.

Don't worry about getting overwhelmed! Just get a function that works for you first and then open a PR and ask questions and I'll help out!

**Comment by vincentarelbundock** [comment]
closed by #968
